app/presenters/activity_csv_presenter.rb

```diff
@@ -113,6 +113,17 @@ def publish_to_iati
     translate("boolean.#{super}")
   end
 
+  def sustainable_development_goals
+    if sdgs_apply == false && step_is_complete_or_next?(activity: self, step: :sustainable_development_goals)
+      "Not applicable"
```

Comment:
There isn't a translation for this; there wasn't one in the base class either. I could elide this by saying `super`, or I could introduce it Somewhere, but there's already an `N/A` localised string, and didn't want to overload that

Reply:
I believe it's because we need to leave nil alone; sdgs_apply is a straight DB field. Nonetheless, I've cargo-culted it because it shouldn't appear to be different – or behave differently from – ActivityPresenter and it'd be more time to work out why and fix in a class that's unrelated to this PR.